topostats/plottingfuncs.py

```diff
@@ -201,6 +204,7 @@ def save_figure(self):
                 vmin=self.zrange[0],
                 vmax=self.zrange[1],
             )
+            # If masked array, apply that on top of the plot
```

Comment:
Nice, it didn't occur to me that virtually nothing needs doing to the `Images()` class as the traces are in essence just refined/slimmed down grain masks and so the functionality to plot them over the image is already there.